hydromt/io/writers.py

```diff
@@ -92,32 +53,15 @@ def configwrite(config_fn: Union[str, Path], cfdict: dict, **kwargs) -> None:
         function.
     """
     root = Path(dirname(config_fn))
-    _cfdict = parse_relpath(cfdict.copy(), root)
+    _cfdict = make_config_paths_relative(cfdict.copy(), root)
     ext = splitext(config_fn)[-1].strip()
     if ext in [".yaml", ".yml"]:
-        _cfdict = _process_config_out(_cfdict)  # should not be done for ini
         with open(config_fn, "w") as f:
             yaml.dump(_cfdict, f, sort_keys=False)
-    elif ext == ".toml":  # user defined
-        _cfdict = _process_config_out(_cfdict)
-        with open(config_fn, "wb") as f:
-            dump_toml(_cfdict, f)
     else:
-        write_ini_config(config_fn, _cfdict, **kwargs)
-
-
-def _process_config_out(d):
-    ret = {}
-    if isinstance(d, dict):
-        for k, v in d.items():
-            if v is None:
-                ret[k] = "NONE"
-            else:
-                ret[k] = _process_config_out(v)
-    else:
-        ret = d
-
-    return ret
+        raise ValueError(
+            f"Could not write to unknown extention: {ext} hydromt only supports yaml"
```

Comment:
```suggestion
            f"Could not write to unknown extension: {ext} hydromt only supports yaml"
```

Reply:
I would opt to add spell checker in your visual studio code extensions.

Reply:
I don't use visual studio code, but I'll see if I can find one for my editor